src/data/preprocess.py

```diff
@@ -0,0 +1,137 @@
+#%%
+import numpy as np
+import matplotlib.pyplot as plt
+import mne
+import time
+import glob
+import pandas as pd
+from pathlib import Path
+from sklearn.preprocessing import MinMaxScaler
+from data_methods import *
+from mne.io import RawArray
+from autoreject import get_rejection_threshold
+#from sklearn.decomposition import FastICA, PCA
+
+
+#%%
+project_dir = str(Path(__file__).resolve().parents[2].joinpath('data/raw/*'))
+file_list_in = glob.glob(project_dir)
+list_im = [file for file in file_list_in]
+list_im.sort()
+print("file_list:{}".format(list_im))
+
+#%%
+# directory to save intermediate/final files to
+save_dir = str(Path(__file__).resolve().parents[2].joinpath('data/processed'))
+
+# store standard montage to populate missing location data (`dig` object)
+montage = mne.channels.make_standard_montage('standard_1020')
+
+# define time step for segmenting data
+tstep = 4.
+
+
+# create info objects
+info = mne.create_info(ch_names=['Fp1','Fp2','F7','F3','Fz','F4','F8','T7',
+                             'C3','Cz','C4','T8','P7','P3','Pz','P4','P8',
+                             'O1','O2','A2', '23A', '24A'],
+                    sfreq=256., ch_types='eeg')
+# set montage of info object
+info.set_montage(montage, on_missing = 'warn')
+
+info2 = mne.create_info(ch_names=['Fp1','Fp2','F7','F3','Fz','F4','F8','T7',
+                             'C3','Cz','C4','T8','P7','P3','Pz','P4','P8',
+                             'O1','O2','A2'],
+                    sfreq=256., ch_types='eeg')
+
+info2.set_montage(montage, on_missing = 'warn')    
+    
+for k in list_im:
+    # file name without extension
+    filename = Path(k).stem
+    print(f"Now creating raw (RawArray) for file {k}\n")
+    raw = mne.io.read_raw_edf(k)
+    data = raw.get_data()
+    
+    
+    if(len(data) == 20):
```

Comment:
```suggestion
    if(len(data) == NUM_CHANNELS):
```

This is just a very small thing and not really needed and it's just for your own learning. Usually when you want to use a specifc number like this, you should define as a variable in all caps, in this case `NUM_CHANNELS = 20` and then refernece it. It makes the readability better and makes making updates to the code easier.